R/filter.R

```diff
@@ -42,6 +42,22 @@ new_filter_block <- function(string = "TRUE", ...) {
       moduleServer(
         id,
         function(input, output, session) {
+          # Restore block state
```

Comment:
@christophsax  and @nbenn I went through another approach that does not involve any button click.

Reply:
I also propagated to the mutate. On top of my head, the worst thing that can happen with `all.equal(eval(formals(list(...)$ctor)$string), string)` is the case where people serialise the block with the default values, that is without changing the filter or mutate values. This would trigger the mutate/filter with the default values, which isn't a big deal I think.